test/Engine/ProtoTest/TD/MultiLangTests/RangeExpressions.cs

```diff
@@ -1964,5 +1964,59 @@ public void AlphabetSequenceNegativeTestCases()
 
             thisTest.VerifyRuntimeWarningCount(6);
         }
+
+        [Test]
+        [Category("SmokeTest")]
+        public void AlphabetSequence()
+        {
+            string src = @"a1;a2;a3;a4;a5;a6;
+[Imperative]
```

Comment:
Yup - in an ideal world if everything worked exactly the way it should we wouldn't need another test. However the tests are there for catching cases were we don't get it right :)

So for each language feature we tend to like to have language tests for both Imperative and Associative.